build.xml

```diff
@@ -5,9 +5,9 @@
 	<target name="compile">
 		<mkdir dir="build/classes"/>
 		<mkdir dir="csrc"/>
-		<javac srcdir="src" destdir="build/classes" debug="on" source="1.5" target="1.5" includeantruntime="false"/>
```

Comment:
I've only actually tested this with Oracle's Java 8 VM on both OS X and Windows, which complained about `source="1.5"` (warnings). Maybe it would be better to update this to 1.8 or drop the `source` parameter altogether? I am not an Ant or JVM expert.